### PR TITLE
Fixed some Android Studio warnings

### PR DESCRIPTION
**Pull request by delphvr**
## Purpose / Description
Removing naming convention warnings in android studio.
## Fixes
 Fixes some of #13282

## Approach
Used code inspection to find the warnings and used refactoring.

## How Has This Been Tested?

I had some issues with running the tests, I used a Emulator - Pixel 7 Pro API 34, but even with seeing the app running and being tested it seemed like they we're almost all skipped.

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/jsaddons/TgzPackageExtract.kt [hunk 1]
```diff
@@ -88,17 +88,17 @@ import java.util.zip.GZIPInputStream
...
 class TgzPackageExtract(private val context: Context) {
-    private val GZIP_SIGNATURE = byteArrayOf(0x1f, 0x8b.toByte())
+    private val _gzipSignature = byteArrayOf(0x1f, 0x8b.toByte())
     private var requiredMinSpace: Long = 0
     private var availableSpace: Long = 0
 
-    private val BUFFER = 512
-    private val TOO_BIG_SIZE: Long = 0x6400000 // max size of unzipped data, 100MB
-    private val TOO_MANY_FILES = 1024 // max number of files
+    private val _buffer = 512
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/jsaddons/TgzPackageExtract.kt:95]: Revert all these

**Review by david-allison — CHANGES_REQUESTED** [review]
Sorry about requesting changes on your first PR, and thanks for submitting!

The issue is meant to be over a single file. 

This gives you: 

* A small task to accomplish
* A "quick win" and confidence boost, in that a PR can likely be accepted within the same day or with minor adjustments
* The ability to focus on issues with setup (such as being unable to run tests)

This gives us: 

* An easy and consistent way to onboard new people and ensure that they can build and test the project
* Slow and gradual progress towards a metric, which we can eventually automate
* An easy code review
* Confidence that people are ready to take on larger tasks and they won't self-assign and feel they've bitten off more than they can chew

Due to the PR size, this may take a number of rounds to get right, and it wasn't intended to be like this

We'll get this merged now the work's been done, but to set expectations: things may take a little longer

File: AnkiDroid/src/main/java/com/ichi2/widget/deckpicker/DeckPickerWidgetConfig.kt [hunk 2]
```diff
@@ -68,7 +68,7 @@ class DeckPickerWidgetConfig : AnkiActivity(), DeckSelectionListener, BaseSnackb
     /**
      * Maximum number of decks allowed in the widget.
      */
-    private val MAX_DECKS_ALLOWED = 5
+    private val _maxDecksAllowed = 5
```
[the remark this answers is not in the document]
Comment (delphvr) [reply AnkiDroid/src/main/java/com/ichi2/widget/deckpicker/DeckPickerWidgetConfig.kt]: I put it as a const, but not sure if I did it like it was attended

File: AnkiDroid/src/main/java/com/ichi2/widget/cardanalysis/CardAnalysisWidgetConfig.kt [hunk 3]
```diff
@@ -63,11 +63,11 @@ class CardAnalysisWidgetConfig : AnkiActivity(), DeckSelectionListener, BaseSnac
     /**
      * Maximum number of decks allowed in the widget.
      */
-    private val MAX_DECKS_ALLOWED = 1
+    private val _maxDecksAllowed = 1
```
[the remark this answers is not in the document]
Comment (delphvr) [reply AnkiDroid/src/main/java/com/ichi2/widget/cardanalysis/CardAnalysisWidgetConfig.kt]: I put it as a const, but not sure if I did it like it was attended (same as above)

File: AnkiDroid/src/main/java/com/ichi2/anki/utils/Time.kt [hunk 4]
```diff
@@ -109,29 +109,29 @@ fun roundedTimeSpanUnformatted(context: Context, time_s: Long): String {
  * Similar to Anki anki/utils.py's fmtTimeSpan.
  *
  * @param context The application's environment.
- * @param time_s The time to format, in seconds
+ * @param timeS The time to format, in seconds
  * @return The formatted, localized time string. The time is always a float. E.g. "**27.0** days"
  */
-fun roundedTimeSpan(context: Context, time_s: Long): String {
```
[the remark this answers is not in the document]
Comment (delphvr) [reply AnkiDroid/src/main/java/com/ichi2/anki/utils/Time.kt]: Done, sorry

File: AnkiDroid/src/main/java/com/ichi2/anki/utils/Time.kt [hunk 5]
```diff
@@ -95,12 +95,12 @@ fun remainingTime(context: Context, time_s: Long): String {
  * Similar to Anki anki/utils.py's fmtTimeSpan.
  *
  * @param context The application's environment.
- * @param time_s The time to format, in seconds
+ * @param timeS The time to format, in seconds
  * @return The formatted, localized time string. The time is always a float. E.g. "27.0 days"
  */
-fun roundedTimeSpanUnformatted(context: Context, time_s: Long): String {
+fun roundedTimeSpanUnformatted(context: Context, timeS: Long): String {
```
[the remark this answers is not in the document]
Comment (delphvr) [reply AnkiDroid/src/main/java/com/ichi2/anki/utils/Time.kt]: Done as well

File: AnkiDroid/src/main/java/com/ichi2/anki/jsaddons/TgzPackageExtract.kt (hunk 1 above)
Comment (delphvr) [reply AnkiDroid/src/main/java/com/ichi2/anki/jsaddons/TgzPackageExtract.kt]: I revert everything you asked! Sorry for the extra work

**Review by david-allison — APPROVED** [review]
Looks good, cheers!

**Review by lukstbit — CHANGES_REQUESTED** [review]
Thank you for contributing!
I've left some minor issues to fix and then we could merge this. It would also be helpful if you could [squash the commits](https://stackoverflow.com/questions/6934752/combining-multiple-commits-before-pushing-in-git) to make it easier for us to merge your changes.

**Comment by delphvr** [comment]
Thank you! I squash them like you asked

**Pull request merged**